R/nonmem.R

```diff
@@ -545,63 +551,16 @@ rex::register_shortcuts("babelmixr2")
 }
 
 .rxToNonmemHandleAssignmentOperator <- function(x, ui) {
```

Comment:
This is a large change, I will have to think about if this works.

Reply:
It passes all of the current tests (a slightly different answer) and the new ones I added.

Reply:
As I've been thinking about it more, I wanted to emphasize that this is the most important part of the PR.  (It's why I started down the path that led to a bigger PR than I initially planned, overall.)  It does two things:  simplifies the assignment code (by removing several nested "if" blocks) and it adds the features for additional assignments (bioavailability, lag, rate, duration, and initial conditions).

Reply:
I think it handled additional assignments already, but code simplification is good.